schema.sql

```diff
@@ -74,4 +78,11 @@ CREATE TABLE  visits (
     CONSTRAINT vets_fk
     FOREIGN KEY (vet) REFERENCES vets(id)
     ON DELETE CASCADE
-);
\ No newline at end of file
+);
+
+CREATE INDEX visits_animal_id_index ON visits(animal_id);
+
+CREATE INDEX visits_vet_index ON visits(vet);
+
+CREATE INDEX visits_vet_index ON visits(vet);
```

Comment:
### Optional
- [ ] It would be better for performance if we add indexes for visits and owners table so we can make queries faster

```
   CREATE INDEX ON visits (animal_id);
   CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);

```